### ISSUE TITLE
Crash in ATLMessageInputToolbar

### ISSUE THREAD

**Issue by mnshah** [layerhq/Atlas-iOS#885]
We are seeing a crash when trying to add an image into a chat screen:

[ATLMessageInputToolbar insertMediaAttachment:](ATLMessageInputToolbar.m:203)

Stack Trace
_________________________________
0	  CoreFoundation                     	0x00000001820702d8 __exceptionPreprocess + 128
1	  libobjc.A.dylib                    	0x00000001938940e4 objc_exception_throw + 56
2	  CoreFoundation                     	0x0000000182070218 +[NSException raise:format:] + 124
3	  Foundation                         	0x0000000182ea9fb4 -[NSConcreteMutableAttributedString addAttribute:value:range:] + 148
4	  PeerWell                           	0x0000000100145730 -[ATLMessageInputToolbar insertMediaAttachment:] (ATLMessageInputToolbar.m:203)
5	  PeerWell                           	0x000000010013358c -[ATLConversationViewController imagePickerController:didFinishPickingMediaWithInfo:] (ATLConversationViewController.m:696)
6	  PhotoLibrary                       	0x000000018d8841c4 PLNotifyImagePickerOfImageAvailability + 84
7	  PhotosUI                           	0x00000001862242d4 -[PUUIImagePickerControllerHelper _notifyImagePickerOfAssetAvailability:] + 172
8	  PhotosUI                           	0x0000000186224068 __71-[PUUIImagePickerControllerHelper handleSelectionOfAsset:inCollection:]_block_invoke69 + 44
9	  PhotosUI                           	0x00000001862c1abc -[PUPhotosGridDownloadHelper _downloadForRequest:didCompleteWithSuccess:canceled:error:] + 316
10	 PhotosUI                           	0x00000001862c1458 __84-[PUPhotosGridDownloadHelper handleDownloadOfAsset:inCollection:withSuccessHandler:]_block_invoke + 100
11	 libdispatch.dylib                  	0x0000000193ee5994 _dispatch_call_block_and_release + 20
12	 libdispatch.dylib                  	0x0000000193ee5954 _dispatch_client_callout + 12
13	 libdispatch.dylib                  	0x0000000193eea20c _dispatch_main_queue_callback_4CF + 1604
14	 CoreFoundation                     	0x00000001820277f8 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 8
15	 CoreFoundation                     	0x00000001820258a0 __CFRunLoopRun + 1488
16	 CoreFoundation                     	0x0000000181f512d4 CFRunLoopRunSpecific + 392
17	 GraphicsServices                   	0x000000018b7676fc GSEventRunModal + 164
18	 UIKit                              	0x0000000186b16fac UIApplicationMain + 1484
19	 PeerWell                           	0x00000001000472e4 main (main.m:14)
20	 libdyld.dylib                      	0x0000000193f12a08 start + 0

**Comment by thomasgpeng** [comment]
@mnshah This was broken in 1.0.9 and should be fixed in 1.0.10.

**Comment by mnshah** [comment]
I've updated to 1.0.10 and image sharing works. However, the location button is unresponsive. Is this a separate issue with 1.0.10?

**Comment by mnshah** [comment]
Also, it looks like the photos do send but in the other person's chat window, they just see a gray box.

**Comment by kmahal** [comment]
@mnshah
1. As @thomasgpeng mentioned the image sharing was fixed in `1.0.10`.  
2. The location sharing is functional, please confirm you have given the application location sharing permission in your phone settings.
3. You need to set the `autoDownloadMIMETypes` of your `LYRClient` instance.  For example:
```objc
    client.autodownloadMIMETypes = [NSSet setWithObjects:ATLMIMETypeTextPlain, ATLMIMETypeImageJPEGPreview, nil];
```
